packages/slots/components/MainFooter.tsx

```diff
@@ -0,0 +1,38 @@
+import { SlotComponentProps } from '../SlotRenderer';
+import { Container, Link } from '@plone/components';
+import Logo from './Logo';
+
+const Footer = (props: SlotComponentProps) => {
+  const { content, location } = props;
+  const siteActions = content?.['@components']?.actions?.site_actions || [];
+
+  return (
+    <Container layout className="footer">
+      {/* TODO: i18n properly */}
+      <div className="footer-message">
+        The Plone® Open Source CMS/WCM is © 2000-2024 by the Plone Foundation
+        and friends. <br />
+        Distributed under the GNU GPL license.
```

Comment:
Now that Volto is distributed under the MIT license, should this be modified to include that? This might be a lawyer or Board question. At the very least, we should specify the license version number.

```suggestion
        Distributed under the GNU GPLv2 license.
```

Reply:
Should we maybe try to have everything under the same license in the plone github org? Agree this is a lawyer/Board question

Reply:
I really don't know. This is the only information I have, but it seems that the Board wants only GPLv2.

https://plone.org/foundation/copyright-licensing-logo/license-faq

Side note, I don't know the process or legal concerns about switching Volto core's license. Contributions have been made under multiple licenses, including BSD2, BSD3, and MIT.

Reply:
I don’t recall a decision that GPL2 was the only license the board wanted, and it was known that React and newer front end packages typically did not use GPL2.

Reply:
Same, should go out, only relevant in demo instances and plone.org, should not be in the default theme.